packages/components/src/components/badge/component.tsx

```diff
@@ -112,6 +113,21 @@ export class KolBadge implements KoliBriBadgeProps {
 		});
 	}
 
+	@Watch('_hideLabel')
+	public validateHideLabel(value?: HideLabelPropType): void {
+		validateHideLabel(this, value);
+	}
+
+	@Watch('_icon')
```

Comment:
Die neuen Watches können raus, weil die Validierung von der Sub-Komponente `kol-span` übernommen wird und in dieser Komponenten diese Pros keine Anwendung finden.